crates/core/component/ibc/src/component/msg_handler/channel_open_confirm.rs

```diff
@@ -65,6 +65,7 @@ impl MsgHandler for MsgChannelOpenConfirm {
             remote: expected_counterparty,
             connection_hops: expected_connection_hops,
             version: channel.version.clone(),
+            ..ChannelEnd::default()
```

Comment:
`ChannelEnd` now has a default impl - so I used it for all new fields. Could set them explicitly?